ecommerce/settings/_oscar.py

```diff
@@ -130,6 +130,8 @@
     'ecommerce.extensions.payment.processors.cybersource.CybersourceREST',
     'ecommerce.extensions.payment.processors.paypal.Paypal',
     'ecommerce.extensions.payment.processors.stripe.Stripe',
+    'ecommerce_extensions.payment.processors.fomopay.Fomopay',
```

Comment:
aren't we adding fomopay here? https://github.com/eduNEXT/edunext-ecommerce/pull/73/ and the urlpatterns

Reply:
This were leftovers from the rebase and deployment test. Now that #73 is merged it should show the correct diff.